test/camelCase.test.js

```diff
@@ -9,5 +9,5 @@ test('includes forward slashes in words', () => {
 });
 
 test('does nothing to an already camel-cased action type', () => {
-  expect(camelCase('myAction')).toBe('myAction');
+  expect(camelCase('myAction')).toBe('myaction');
```

Comment:
Please fix this behaviour, because it shouldn't do anything to the string according to the spec.